### ISSUE TITLE
PostCSS plugin is not assigned when PostCSS plugins is typeof Array

### ISSUE THREAD

**Issue by andrewvasilchuk** [nuxt-community/nuxt-tailwindcss#34]
I am getting an error, because previously in my Nuxt.js project (scaffolded with `create-nuxt-app`) I have `build.postcss` which type is `Array`. I googled a little and found out that this syntax is deprecated, and you should use object notation instead, like stated in [docs](https://nuxtjs.org/faq/postcss-plugins).

I skimmed through [`module.js`](https://github.com/nuxt-community/nuxt-tailwindcss/blob/master/lib/module.js#L35) and found the following line of code:

https://github.com/nuxt-community/nuxt-tailwindcss/blob/004fc2d755eeac9194c0441c51e076be8bda71fb/lib/module.js#L34

I think it would be nice if you can support deprecated syntax, so users do not get sneaky errors.

**Comment by andrewvasilchuk** [comment]
Maybe solution can look something like this:
```js
const { plugins } = this.options.build.postcss

this.options.build.postcss.preset.stage = 1

if (typeof plugins === 'object' || plugins === undefined) {
  this.options.build.postcss.plugins = this.options.build.postcss.plugins || {}
  this.options.build.postcss.plugins.tailwindcss = this.options.build.postcss.plugins.tailwindcss || configPath
} else if (Array.isArray(plugins)) {
  plugins.push(require('tailwindcss')(plugins.tailwindcss || configPath))
}
```

**Comment by Atinux** [comment]
Hi @andrewvasilchuk 

I think it's possible yes, actually it should be quite easy, I don't have much time right now but will be happy to review a PR :)

**Comment by Atinux** [comment]
Released in `v1.2.0`
